WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductRowCard.swift

```diff
@@ -5,15 +5,18 @@ struct CollapsibleProductRowCard: View {
     @ObservedObject var viewModel: ProductRowViewModel
     @State private var isCollapsed: Bool = true
 
+    var onRemoveProduct: () -> Void
+
```

Comment:
Could we use `viewModel.removeProductIntent()` instead?

Reply:
Oh! Much simpler 😅 Changed: https://github.com/woocommerce/woocommerce-ios/pull/10861/commits/b16620eb992a3e690a3da6e3de323f171eb4d8c5